package.json

```diff
@@ -39,9 +39,9 @@
     "yup": "^0.32.11"
   },
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3000",
```

Comment:
not necessaray anymore can be dynamic if your .env is setup with the same NEXT_PUBLIC_ROOT_DOMAIN